demo/dep/libtmux/default.nix

```diff
@@ -0,0 +1,2 @@
+# DO NOT HAND-EDIT THIS FILE
```

Comment:
How else? Without instructions how to maintain this, we are not in a position to approve this change.

Reply:
nix-thunk generates this, it doesn't have to be how libtmux is pinned (using a newer nixpkgs was discussed for example).